core/src/com/unciv/logic/city/CityReligion.kt

```diff
@@ -214,4 +215,24 @@ class CityInfoReligionManager {
         
         updateNumberOfFollowers()
     }
+    
+    /** Doesn't update the pressures, only returns what they are if the update were to happen right now */
+    fun getPressuresFromSurroundingCities(): Counter<String> {
+        val addedPressure = Counter<String>()
+        if (cityInfo.isHolyCity()) {
+            addedPressure[religionThisIsTheHolyCityOf!!] = 5 * pressureFromAdjacentCities
+        }
```

Comment:
I highly doubt this is how it works. If I'm a holy city I have 5x the *existing* pressure as pressure *added*?

Reply:
`pressureFromAdjacentCities` is a cached variable of the pressure a single city adds to adjacent cities (excluding itself). Holy cities do pressure themselves, and 5 times as much as adjacent cities pressure each other, so this is implemented correctly

Reply:
Huh
Ok then